src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -19,9 +22,14 @@
   private final static String CLIENT_SECRET = "91fd789bf568ec43d2ee";
   private final static String PUSHER_APPLICATION_NAME = "song-guessing-game";
   private final static String PUSHER_CHAT_CHANNEL_NAME = "chat-update";
+  private final static Type MESSAGE_TYPE = new TypeToken<Map<String, String>>(){}.getType();
   private Pusher pusher;
   private Gson gson;
 
+  // TODO: @salilnadkarni remove temp variables and integrate datastore
+  Map<String, Boolean> status  = new HashMap<String,Boolean>();
```

Comment:
Note: "status" is a little vague for a variable name. Looking through your code, it looks like this is a map from a user ID to whether their guess was correct, so I there are a few options for how to name this variable:
- guessCorrectness
- guessCorrectnessStatus
- userIdToGuessCorrectnessStatus
- userIdToGuessCorrectness

I'd say let's pick one of those to be more specific and also update related variable names. In addition, we'll want to use similar specific names for the datastore properties or entities we wind up using.